tap_quickbooks/discover.py

```diff
@@ -29,6 +31,18 @@ def _load_schemas():
 
     return schemas
 
+def _load_shared_schema_refs():
```

Comment:
Why did we remove the older load_schema and added a new one? Do add function comments with details.

Reply:
Initially, there were only 2 schema refs, hence, it was handled manually in the `do_discover` function. Now as, we have some reference schemas, thus, created a separate function to prepare schema refs.